server/ansible/roles/pbench-server-activate-setup-results-host-info/tasks/main.yml

```diff
@@ -27,10 +31,11 @@
   with_items:
     - "002"
 
-- name: "set up links to the active files"
+- name: "set up links to the maintenance files"
   file:
     path: "{{ httpd_document_root_dir }}/{{ pbench_host_info_prefix }}{{ item }}"
-    src:  "{{ pbench_host_info_prefix | basename }}{{ item }}.active"
+    src:  "{{ pbench_host_info_prefix | basename }}{{ item }}.maint"
```

Comment:
So, is this creating a symlink to the file created at lines 25-28?

I note that that file is set up without specifying a `mode:` or `group:` (unlike lines 19-20)...that seems like it could be a problem (i.e., if the message set up at line 27 is supposed to be widely accessible...).

Reply:
Typically this is not a problem because symlinks are created with wide open permissions by default.

@webbnh, can you file an issue to track this separately?